### ISSUE TITLE
Entitys not running on server!

### ISSUE THREAD

**Issue by SmellyModder** [HarryTechRevs/MinecraftModding#8]
It appears your mobs in your tutorials do not run on servers!

**Comment by HoneyFun** [comment]
Like, it crashes during server state CONSTRUCTION?

**Comment by SmellyModder** [comment]
NoClassDefFoundError for the server

**Comment by HoneyFun** [comment]
How did you get it to work? (I have been having the same problem and have had the same exception)

**Comment by SmellyModder** [comment]
So I made a seperate preInit for side only 

        @SideOnly(Side.CLIENT)
	@EventHandler
	public static void preinitSide(FMLPreInitializationEvent event)
	{
		RegistryHandler.preInitRegistriesSide();
	}

Then In my registry handler
@SideOnly(Side.CLIENT)
public static void preInitRegistriesSide() {
		
		RenderHandler.registerEntityRenders();
}

**Comment by HoneyFun** [comment]
Ah. Thanks so much!

**Comment by SmellyModder** [comment]
Are you harry?

**Comment by HoneyFun** [comment]
No, I am someone who has been looking through harry's tutorials and following them.

**Comment by SmellyModder** [comment]
Okay always looking to help people with stupid tutorial bugs.
